src/data-loader.ts

```
@@ -135,6 +135,22 @@ export function getClaudePaths(): string[] {
 	return paths;
 }
 
+/**
+ * Get context usage percentage thresholds for color coding
+ * Can be configured via environment variables or uses defaults
+ * @returns Context usage thresholds with LOW and MEDIUM percentages
+ */
+export function getContextUsageThresholds(): { readonly LOW: number; readonly MEDIUM: number } {
+	// Check environment variables with fallback to defaults
+	const lowThreshold = process.env[CONTEXT_LOW_THRESHOLD_ENV];
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)